_posts/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1.md

```diff
@@ -21,7 +21,186 @@ sidebar:
+
+
+
+{:.output}
+    33.8
+
+
+
+### Question 7 - Run Calculations on Variables (10 pts)
+
+Add a new code cell below, and add `Python` code to:
+
+* Convert the variable values from Fahrenheit to Celsius using the following equation:
+    * Celsius = (Fahrenheit - 32) / 1.8
+    * Note that including `Fahrenheit - 32` within parenthesis `()` tells `Python` to execute that calculation first. 
+* Print the new variable for December. 
+
+
+{:.output}
+    0.9999999999999984
+
+
+
+### Question 8 - Create Lists (6 pts)
+
+Add a new code cell below, and add `Python` code to:
+
+* Create a new list for average monthly temperature for Boulder, CO from the variables containing the Celsius values. 
+* Print the temperature list.
+* Create another list for the month names using **full-length** month names (e.g. January).
+* Print the month name list.
+
+
+{:.output}
+    [0.0, 2.000000000000001, 5.0, 9.555555555555557, 14.388888888888888, 19.555555555555557, 22.77777777777778, 21.722222222222218, 16.72222222222222, 11.61111111111111, 4.8888888888888875, 0.9999999999999984]
+    ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+
+
+
+### Question 9 - Plot Data From Lists (12 pts)
+
+Add a new code cell below, and add `Python` code to:
+
+* Create a **red bar** plot of the average monthly temperature for Boulder, CO using the lists created in Question 5. 
+* Be sure to include a title for your plot.
+* Be sure to include labels for the x- and y-axes, **including the units for the temperature**.
+* Rotate the x-axis markers, so that the spacing along the x-axis is more appealing with long month names.
+
+
+{:.output}
+{:.display_data}
+
+![png]({{ site.url }}//images/courses/earth-analytics-bootcamp/00-course-overview/2018-07-16-co-04-homework-1_8_0.png)
+
+
+
+
+### Question 10 - Discuss Plot (4 pts)
+
+Add a new `Markdown` cell below, and answer the following questions (4-6 sentences).
+
+1. Which months have the lowest and the highest temperatures? How much of a difference is there between the lowest and highest temperatures?
+
+2. Given that Boulder, CO is in the Northern Hemisphere, are these results surprising? How might the results look different if you were plotting data from a location in the Southern Hemisphere? 
+
+3. Given the Boulder, CO is at a moderately high latitude (~40 degrees North), are these results surprising? How might the results look different if you were plotting data from a location closer to the equator? (Hint: think about the difference between the lowest and highest temperatures)
+
+To answer these questions, it can help to review the <a href="https://www.google.com/maps/place/Boulder,+CO/@18.3736306,-81.2965677,3.85z/data=!4m5!3m4!1s0x876b8d4e278dafd3:0xc8393b7ca01b8058!8m2!3d40.0149856!4d-105.2705456" target = "_blank">location of Boulder Colorado on Google Maps</a>. 
+
+
+## Submit Part 2 - Homework 1 Using Git/Github.com
```

Comment:
This is a bit confusing 
how about 
## Part 3 - Submit Jupyter Notebook to Github xxx repo 

submit part 2 is confusing .